src/precice/tests/ParallelTests.cpp

```diff
@@ -867,6 +867,248 @@ BOOST_AUTO_TEST_CASE(TestBoundingBoxInitialization)
   precice.finalize();
 }
 
+// StartIndex is here the first index to be used for writing on the slave rank
+void runTestAccessReceivedMesh(const std::string         configName,
+                               const std::vector<double> boundingBoxSlave,
+                               const std::vector<double> writeDataSlave,
+                               const std::vector<double> expectedPositionSlave,
+                               const std::vector<double> expectedReadDataSlave,
+                               const int                 startIndex)
+{
+
+  PRECICE_TEST("SolverOne"_on(2_ranks), "SolverTwo"_on(2_ranks));
```

Comment:
Wouldn't we be able to test for edge case at once if we use a 3-1 distribution?

Reply:
I don't really understand which edge cases you have in mind.

Reply:
Nothing specific. We would just have more flexibility:
- One rank empty, the other two overlapping
- Three ranks overlapping

Maybe these cases are not so relevant.